### ISSUE TITLE
Can't run script that runs another script found in its $PSScriptRoot

### ISSUE THREAD

**Issue by sba923** [IISResetMe/MeasureScript#5]
Consider this `Parent.ps1`:

````
$script = Join-Path -Path $PSScriptRoot -ChildPath 'Child.ps1'
& $script -arg 'somearg'
````

that invokes this `Child.ps1`:

````
param([string] $arg)
Write-Host("Child.ps1 called with '-arg $arg'")
````

Running `Parent.ps1` (expectedly) yields:

````
Child.ps1 called with '-arg somearg'
````

but `Measure-Script -Path Parent.ps1` fails:

````
Measure-Script : Cannot bind argument to parameter 'Path' because it is an empty string.
At line:1 char:1
+ Measure-Script -path ../Parent.ps1
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidData: (:) [Measure-Script], ParameterBindingValidationException
    + FullyQualifiedErrorId : ParameterArgumentValidationErrorEmptyStringNotAllowed,Measure-Script
````

**Comment by IISResetMe** [comment]
Thanks for reporting this @sba923! 

This is due to how we "rewrite" the AST in-memory, so by the time we execute it, it's no longer actually a script. 

We might be able to inject meaningful `$PSScriptRoot/$PSCommandPath` vars, but `$MyInvocation` will still not reflect the original script (just FYI)

**Comment by IISResetMe** [comment]
This is fixed in #8
